.pre-commit-config.yaml

```diff
@@ -59,8 +59,9 @@ repos:
       - id: mixed-line-ending
         exclude: \.(cs|xml)$
       - id: trailing-whitespace
-        files: \.(in|java|md|py|rb|sh|vue|yaml|yml)$
+        files: \.(header|in|java|md|properties|py|rb|sh|sql|txt|vue|yaml|yml)$
```

Comment:
any .header files?

Reply:
At least one 

https://github.com/apache/cloudstack/blob/main/LICENSE.header

Reply:
ah, ok, it wasn't mentioned in this PR, hence my question